address_book/cmd/server/main.go

```diff
@@ -0,0 +1,25 @@
+package main
+
+import (
+	"addressbook/internal/pb"
+	"addressbook/internal/service"
+	"log"
+	"net"
+
+	"google.golang.org/grpc"
+)
+
+func main() {
+
+	lis, err := net.Listen("tcp", ":8080")
```

Comment:
Please don't use hardcoded values, add constant or variable and use it instead.

Reply:
you can use default values and change it via cli or env

for example you can use https://github.com/spf13/viper

Reply:
Done